articles/azure-functions/functions-runtime-install.md

```diff
@@ -22,74 +22,74 @@ ms.author: anwestg
 
 If you would like to install the Azure Functions Runtime preview, you must follow these steps:
 
-1. Ensure your machine passes the minimum requirements
-1. Download the [Azure Functions Runtime Preview Installer](https://aka.ms/azafr). 
-1. Install the Azure Functions Runtime preview
-1. Complete the configuration of the Azure Functions Runtime preview
+1. Ensure your machine passes the minimum requirements.
+1. Download the [Azure Functions Runtime Preview Installer](https://aka.ms/azafr).
+1. Install the Azure Functions Runtime preview.
+1. Complete the configuration of the Azure Functions Runtime preview.
 
 ## Prerequisites
 
 Before you install the Azure Functions Runtime preview, you must have the following:
 
 1. A machine running Microsoft Windows Server 2016 or Microsoft Windows 10 Creators Update (Professional or Enterprise Edition).
-1. A SQL Server instance running within your network.  Minimum edition requirement is SQL Server Express.
+1. A SQL Server instance running within your network.  Minimum edition required is SQL Server Express.
 
 ## Install the Azure Functions Runtime Preview
 
-The Azure Functions Runtime preview installer guides you through the installation of the Azure Functions Runtime preview Management and Worker Roles.  It is possible to install the Management and Worker role on the same machine.  However, as you add more Functions, you must deploy more worker roles on additional machines to be able to scale your functions onto multiple workers.
+The Azure Functions Runtime Preview Installer guides you through the installation of the Azure Functions Runtime preview Management and Worker Roles.  It is possible to install the Management and Worker role on the same machine.  However, as you add more Functions, you must deploy more worker roles on additional machines to be able to scale your functions onto multiple workers.
 
 ## Install the Management and Worker Role on the same machine
 
 1. Run the Azure Functions Runtime Preview Installer.
 
     ![Azure Functions Runtime Preview Installer][1]
 
-1. **Click Next** advance past the first stage of the installer
-1. Once you have read the terms of the **EULA**, **check the box** to accept the terms and **click Next** to advance.
-1. Now select the roles you want to install on this machine **Functions Management Role** and/or **Functions Worker Role** and **Click Next**
+1. Click **Next** advance past the first stage of the setup.
```

Comment:
"Setup Wizard" as per the UI, or better yet, drop everything after "Click Next".